install.sh

```diff
@@ -1,6 +1,14 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
i added a `-e` option here to your shell script to ensure that the script stops if a sigterm is ever received.